test/ng/rootScopeSpec.js

```diff
@@ -1255,6 +1255,55 @@ describe('Scope', function() {
       });
     });
 
+
+    describe('$suspend/$resume', function() {
+      it('should suspend watchers on scope', inject(function($rootScope) {
+        var watchSpy = jasmine.createSpy('watchSpy');
+        $rootScope.$watch(watchSpy);
+        $rootScope.$suspend();
+        $rootScope.$digest();
+        expect(watchSpy).not.toHaveBeenCalled();
+      }));
+
+      it('should resume watchers on scope', inject(function($rootScope) {
+        var watchSpy = jasmine.createSpy('watchSpy');
+        $rootScope.$watch(watchSpy);
+        $rootScope.$suspend();
+        $rootScope.$resume();
+        $rootScope.$digest();
+        expect(watchSpy).toHaveBeenCalled();
+      }));
+
+      it('should suspend watchers on child scope', inject(function($rootScope) {
```

Comment:
Maybe test that `$resume` works in this case as well?

Reply:
See https://github.com/angular/angular.js/pull/16308/files#diff-bab291b2ff3232222e01694707b01a1dR1286